src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.js

```diff
@@ -197,6 +211,16 @@ export const GenericTable = ({
   const isDataFetchedFromDataset = !!parameterData?.options?.dynamicValues;
 
   const _getDataFromTwingraphDataset = async (setClientFileDescriptor) => {
+    const _setClientFileDescriptorToNull = () => {
+      setClientFileDescriptor({
+        file: null,
+        content: null,
+        agGridRows: null,
+        errors: null,
+        tableDataStatus: TABLE_DATA_STATUS.ERROR,
+      });
+    };
```

Comment:
this function is used only once, not sure there is a real benefit to create it :thinking:

Reply:
I thought it was used twice... Maybe a mess after refactor.